src/components/Resources/Schema/SchemasList.tsx

```diff
@@ -197,7 +197,7 @@ const SchemaList = (props: { schemaSelectionCallback: (schemaId: string, schemaD
 									schemaList.map((element, key) => (
 										<div className='p-2' key={key}>
 											<SchemaCard schemaName={element['name']} version={element['version']} schemaId={element['schemaLedgerId']} issuerDid={element['issuerId']} attributes={element['attributes']} created={element['createDateTime']}
-												onClickCallback={schemaSelectionCallback} />
+												onClickCallback={schemaSelectionCallback} className='' />
```

Comment:
remove this empty className

Reply:
Removed. Given optional to handle type error.